src/api/status/public/js/build-log/build-header.js

```diff
@@ -11,17 +12,19 @@ const buildStarted = document.getElementById('build-started');
 const buildDuration = document.getElementById('build-duration');
 const buildPrevious = document.getElementById('previous-build');
 const buildLoadingAnimation = document.getElementById('build-animation');
+const buildCommitMessage = document.getElementById('build-commit-message');
 
 function renderBuildTimeInfo(startedDate, stoppedDate = new Date()) {
   const duration = new Date(stoppedDate).getTime() - new Date(startedDate).getTime();
   buildDuration.innerText = ms(duration);
   buildStarted.innerText = new Date(startedDate).toUTCString();
 }
 
-function renderSender(sender) {
+function renderSender(sender, headCommit) {
   buildSender.href = sender.html_url;
   buildSenderName.innerText = sender.login;
   buildSenderImg.src = sender.avatar_url;
+  buildCommitMessage.innerText = `Commit message: ${headCommit.message}`;
```

Comment:
We can probably drop the `Commit message:` prefix, it's clear what it is, and this will take up space.